### PR TITLE
Leaderboard  Setup Project

### PR DESCRIPTION
**Pull request by looh16**
Project setup for the Leaderboard list app. was used:
1.  webpack
2. ES6 features
3. HTML
4. CSS

### CODE REVIEW

File: src/page.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
+// eslint-disable-next-line
```
Comment (FrederickMih) [inline src/page.js:1]: - [ ] Please avoid disabling Linters. There are great tools that flag programming bugs. Rather, fix them.

File: src/page.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+  const scoreInput = document.createElement('input');
+  scoreInput.setAttribute('type', 'text');
+  scoreInput.setAttribute('name', 'score');
+  scoreInput.setAttribute('placeholder', 'Your Score');
+
+  const submitButton = document.createElement('input');
+  submitButton.className = 'btnSubmit';
+  submitButton.setAttribute('type', 'submit');
+  submitButton.setAttribute('value', 'Submit');
```
Comment (FrederickMih) [inline src/page.js:104]: - [ ] Hi @looh16, based on the requirements of this project, the submit button should hit the POST API endpoint and post user entries. Kindly check the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md) of this project and fix this.

File: src/page.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+  const tableTitle = document.createElement('h2');
+  tableTitle.className = 'tableTitle';
+
+  const tableTitleText = document.createTextNode('Recent score');
+  tableTitle.appendChild(tableTitleText);
+  const tableRefreshButton = document.createElement('button');
+  tableRefreshButton.className = 'refreshBtn';
+
+  tableRefreshButton.innerHTML = 'Refresh';
```
Comment (FrederickMih) [inline src/page.js:72]: - [ ] Again, based on the requirements of this project, when the refresh button is clicked, it should hit the GET API endpoint, causing the scores list to change. You haven't yet implemented this. Let's once more check the general requirements of this project and work on this.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @looh16 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights

- Linters checks are passing :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/page.js (hunk 3 above)
Comment (looh16) [reply src/page.js:72]: Thanks! I made a mistake, this pull request was for a basic setup project, I submitted it by mistake. but I will follow the recommendations and complete what is missing. Thank you!

**Review by FrederickMih — APPROVED** [review]
Hi @looh16 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**